### PR TITLE
Template screen to apply template changes from UI.

### PR DESCRIPTION
**Pull request by jothi-prasad**
This PR introduces a screen for applying templates from UI. if you have dynamic params(Keywords) in your template, then the ui will prompt you to enter them before applying template.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: ui/src/components/template/TemplateDescription.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    display: table-cell;
+    background-color: ${(props) => props.color};
+`;
+
+const TrStyled = styled.tr`
+    background-color: ${(props) => props.color};
+`;
+
+export default class TemplateDescription extends React.Component {
+    constructor(props) {
```
Comment (pratikgote) [inline ui/src/components/template/TemplateDescription.js:42]: can we remove the empty constructor ?

Comment (jothi-prasad) [reply ui/src/components/template/TemplateDescription.js]: Done

File: ui/src/components/template/ApplyTemplate.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        this.state = {
+            keywordsList: [],
+        };
+    }
+
+    componentDidMount() {
+        this.InitialKeywordsLoad();
+    }
+
+    InitialKeywordsLoad(button) {
```
[the remark this answers is not in the document]
Comment (jothi-prasad) [reply ui/src/components/template/ApplyTemplate.js]: done.

File: ui/src/components/denali/icons/Icons.js [hunk 3]
```diff
@@ -127,8 +127,9 @@ export const ICONS = {
...
     ],
     'information-circle': [
-        'M24,2A22,22,0,1,0,46,24,21.9,21.9,0,0,0,24,2Zm0,40A18,18,0,1,1,42,24,18.1,18.1,0,0,1,24,42Z',
-        'M24,20a2,2,0,0,0-2,2V34a2,2,0,0,0,4,0V22A2,2,0,0,0,24,20Z',
+        "M512 42.667c-259.206 0-469.333 210.128-469.333 469.333s210.128 469.333 469.333 469.333c259.206 0 469.333-210.128 469.333-469.333v0c0.003-0.635 0.005-1.387 0.005-2.138 0-258.027-209.173-467.2-467.2-467.2-0.752 0-1.504 0.002-2.255 0.005l0.117-0zM512 896c-212.077 0-384-171.923-384-384s171.923-384 384-384c212.077 0 384 171.923 384 384v0c-1.203 211.592-172.408 382.797-383.885 383.999l-0.115 0.001z",
```
[the remark this answers is not in the document]
Comment (jothi-prasad) [reply ui/src/components/denali/icons/Icons.js]: done.

**Pull request merged**